src/manifold/src/impl.cpp

```diff
@@ -655,25 +661,24 @@ void Manifold::Impl::IncrementMeshIDs(int start, int length) {
   ASSERT(start >= 0 && length >= 0 && start + length <= triBary.size(),
          logicErr, "out of bounds");
   const auto policy = autoPolicy(length);
-  // Use double the space since the Boolean has P and Q instances.
-  VecDH<int> includesMeshID(2 * Manifold::Impl::meshIDCounter_, 0);
+  HashTable<uint32_t> meshidTable(std::max(16u, meshids * 2));
```

Comment:
Is this hash table really going to be big enough to warrant trying to use a GPU-accelerated one? Would a regular STL hash table be simpler?

Reply:
Also, shouldn't this be `min` instead of `max`? Or else aren't we allocating just as much memory as the vector was?

Reply:
The hash table itself is not large enough, but we can have many vertices that we can benefit from using GPU acceleration. 

The max is used to avoid allocating a hash table that is too small at the start, which may cause slowness in the GPU implementation due to warp divergence. Although I am not sure if this is really needed, I added it while debugging meshids (it was set to some weird values previously but the bug is fixed)